base/src/main/java/org/aya/core/def/CtorDef.java

```diff
@@ -52,11 +52,7 @@ public CtorDef(
     return new DataDef.CtorTelescopes(dataSignature, conSignature.param());
   }
 
-  @Override public <P, R> R accept(@NotNull Visitor<P, R> visitor, P p) {
-    return visitor.visitCtor(this, p);
-  }
-
-  public @NotNull DefVar<CtorDef, TeleDecl.DataCtor> ref() {
+    public @NotNull DefVar<CtorDef, TeleDecl.DataCtor> ref() {
     return ref;
   }
```

Comment:
I think it's because of some stupid code generation bugs. IDEA should auto-format the generated code, but sometimes it doesn't. I guess @wsx-ucb just needs to configure a pre-commit formatting checkbox ;p